src/main/java/com/gluonhq/substrate/target/WindowsTargetConfiguration.java

```diff
@@ -285,6 +285,7 @@ private void createIconResource() throws InterruptedException, IOException {
     // During development if user changes the application icon, the same is not reflected immediately in Explorer.
     // To fix this, a cache clearance of the Windows explorer is required.
     private void clearExplorerCache() throws IOException, InterruptedException {
+        if (!executableOnPath("ie4uinit.exe")) return;
```

Comment:
Add a log if this happens, warning the user that the application icon might have been cached and not updated properly?